lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_converter.hpp

```diff
@@ -85,11 +82,11 @@ class S1apStateConverter : StateConverter {
   static void proto_to_enb(const oai::EnbDescription& proto,
                            enb_description_t* enb);
 
-  static void ue_to_proto(const ue_description_t* ue,
+  static void ue_to_proto(const oai::UeDescription* ue,
                           oai::UeDescription* proto);
 
   static void proto_to_ue(const oai::UeDescription& proto,
```

Comment:
Remove this function

Reply:
https://github.com/magma/magma/pull/13678#discussion_r975489492